jenkins/pipelines/llnode-pipeline.jenkinsfile

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env groovy
```

Comment:
I'm ok with landing so we can iterate but would it be useful to have a comment that indicates "Not yet working" just in case somebody without the context takes a look ?

Reply:
I think these scripts will work now since tests are working on llnode with Travis.